parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -644,18 +644,23 @@ users:
     set -x
 }
 
-ensureRunCommandCompleted
-echo `date`,`hostname`, RunCmdCompleted>>/opt/m
+if [[ "$CONTAINER_RUNTIME" == "clear-containers" ]]; then
+	# If the container runtime is "clear-containers" we need to ensure the
+	# run command is completed _before_ we start installing all the dependencies
+	# for clear-containers to make sure there is not a dpkg lock.
+	ensureRunCommandCompleted
+	echo `date`,`hostname`, RunCmdCompleted>>/opt/m
+fi
 
 if [[ $OS == $UBUNTU_OS_NAME ]]; then
```

Comment:
Is there a reason why this got moved up in your previous PR? if not it can probably go back to right after ```echo `date`,`hostname`,` RunCmdCompleted>>/opt/m```

Reply:
I moved it up since we are installing things for clearcontainers i didn't want it to upgrade... but can move inside the `if` if that hellps

Reply:
Sounds good. I think it's needed for all runtimes, not just clearcontainers so let's leave it as is